sched/signal/sig_timedwait.c

```diff
@@ -260,8 +261,9 @@ int nxsig_timedwait(FAR const sigset_t *set, FAR struct siginfo *info,
    * signals in the pending signal set argument.
    */
 
-  intersection = *set & nxsig_pendingset(rtcb);
-  if (intersection != NULL_SIGNAL_SET)
+  pendingset = nxsig_pendingset(rtcb);
```

Comment:
`pendingset` is not really needed as `intersection` can be re-used. Probably added just for readability

Reply:
It is getting late here and will be shutting things down.  I will catch up with you tomorrow.

Reply:
Done!